### DISCUSSION TITLE
Cannot connect to Polygon

### DISCUSSION THREAD

**Discussion by wpitallo** [WalletConnect/walletconnect-monorepo#758] [Developer Support] [answered]
Feature Request:

Would be great to add support for Polgon Network, I have tried to connect but to no avail. I get connected but network chain is always mainnet.

**Comment by romain130492** [comment]
any news about this?.. Polygon is getting quite big.

**Comment by chadyj** [comment, marked as answer]
Polygon is supported in v2.
See what else is new here https://medium.com/walletconnect/walletconnect-v2-0-protocol-whats-new-3243fa80d312
